tests/e2e/common-actions/memory-efficiency-actions.ts

```diff
@@ -0,0 +1,29 @@
+import {t} from 'testcafe';
+import { MemoryEfficiencyPage } from '../pageObjects';
+
+const memoryEfficiencyPage = new MemoryEfficiencyPage();
+export class MemoryEfficiencyActions {
+    async voteForVeryUsefulAndVerifyDisabled(): Promise<void> {
```

Comment:
just need a description for each method here
like:
    /**
     * Check that voted very useful is disabled
     */